### DISCUSSION TITLE
brightness adjiustment from external ipad sensor to desktop monitor in sidecar mode

### DISCUSSION THREAD

**Discussion by quincymondragon** [MonitorControl/MonitorControl#1079] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

feature request no problem related

### Describe the solution you'd like

Is possible to use apple ipad brightness sensor to adjust brightness desktop monitor during sidecar mode?

### Describe alternatives you've considered

In alternative is possible to use ipad brightness settings to set brightness monitor

**Comment by waydabber** [comment]
I was looking into this one before but there is no simple way to do this. Maybe a background helper app running on iPad, but that is certainly out of scope I am afraid.
